docs/linux/setup_ubuntu-host_qemu-vm_x86-64-kernel.md

```diff
@@ -30,6 +30,9 @@ Generate default configs:
 cd $KERNEL
 make CC="$GCC/bin/gcc" defconfig
 make CC="$GCC/bin/gcc" kvmconfig
+# After Linux kernel v5.10
+make CC="$GCC/bin/gcc" defconfig
+make CC="$GCC/bin/gcc" kvm_guest.config
 ```
```

Comment:
I would do:

``` bash
cd $KERNEL
make CC="$GCC/bin/gcc" defconfig
# Before Linux kernel v5.10:
make CC="$GCC/bin/gcc" kvmconfig
# After Linux kernel v5.10:
make CC="$GCC/bin/gcc" kvm_guest.config
```

This way there's less duplication and it's clearer to which line the comment relates.

Reply:
You can `git checkout v5.0 && make kvm_guest.config`, `git checkout v4.14 && make kvm_guest.config`, etc to find the oldest release that supports it.